### PR TITLE
cmake: vkcube/vulkaninfo successfully compile for Android

### PR DESCRIPTION
**Pull request by juan-lunarg**

### CODE REVIEW

File: cube/CMakeLists.txt [hunk 1]
```diff
@@ -171,40 +171,53 @@ endif()
...
-        target_link_libraries(vkcube rt)
+        target_link_libraries(vkcube PRIVATE rt)
     endif()
-
-    if (ENABLE_ADDRESS_SANITIZER)
-        target_compile_options(vkcube PUBLIC -fsanitize=address)
-        target_link_options(vkcube PUBLIC -fsanitize=address)
-    endif ()
+    target_link_libraries(vkcube PRIVATE Vulkan::Headers Vulkan::Loader)
 else()
```
Comment (charles-lunarg) [inline cube/CMakeLists.txt:199]:
```suggestion
elseif(WIN32)
```

Comment (charles-lunarg) [reply cube/CMakeLists.txt:199]: Might as well make it specific to WIN32 - in case some other platform wants to be added in the future.

**Comment by juan-lunarg** [comment]
Not sure why but MacOS started complaining about sprintf...

```
/Users/runner/work/Vulkan-Tools/Vulkan-Tools/cube/cube.c:529:5: error: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Werror,-Wdeprecated-declarations]
    sprintf(message, "%s - Message Id Number: %d | Message Id Name: %s\n\t%s\n", prefix, pCallbackData->messageIdNumber,
```

File: cube/CMakeLists.txt (hunk 1 above)
Comment (juan-lunarg) [reply cube/CMakeLists.txt]: 👍🏾

**Review by charles-lunarg — APPROVED** [review]
I'm good with these changes - assuming CI passes of course. Android is a complex beast so I commend you for tackling it.

**Pull request merged**